docs/howtos/extending/magicgui.md

```diff
@@ -102,17 +101,13 @@ widget_demo.show()
 the appropriate widget type for a given function parameter, and to indicate a
 context-dependent action for the object returned from the function (in the
 absence of a type hint, the type of the default value will be used). You can also
-customize your widget using
-[`magicgui.magicgui`](https://pyapp-kit.github.io/magicgui/api/magicgui/#magicgui.magicgui)
+customize your widget using {func}`magicgui.magicgui`
 parameters. In the example above, `call_button` specifies the button text and the
 `param_options` `slider_float` and `dropdown` let you customize the widget
 associated with those function parameters.
 
 Third party packages (like napari in this case) may provide support for their types
-using
-[`magicgui.register_type`](https://pyapp-kit.github.io/magicgui/usage/types_widgets.html#register-type).
-Indeed napari uses
-[`magicgui.register_type`](https://pyapp-kit.github.io/magicgui/usage/types_widgets.html#register-type)
+using {func}`magicgui.register_type`. Indeed napari uses {func}`magicgui.register_type`
```

Comment:
These links seem to not be resolving, I think we need
```suggestion
using {func}`magicgui.type_map.register_type`. Indeed napari uses {func}`magicgui.type_map.register_type`
```

Reply:
Thanks, sorry I forgot to check the links. I was trying to use the same object path as I saw in magicgui docs (see [here](https://github.com/pyapp-kit/magicgui/blob/1e78e6f0bee37aeb9ce7d300a52f5f120141f834/docs/type_map.md?plain=1#L273)) but Talley must use different API auto doc thing for magicgui.